### PR TITLE
Base functionalyty

### PR DESCRIPTION
**Pull request by mociepka**

### CODE REVIEW

File: i18naddress/downloader.py [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+def fetch(url):
+    logger.debug(url)
+    data = requests.get(url).json()
+    return data
+
+
+def process(key):
+    url = '%s/%s' % (MAIN_URL, key)
+    data = fetch(url)
+    for unwanted in [
```
Comment (patrys) [inline i18naddress/downloader.py]: Please remove this filtering and regenerate the JSON files so they are complete.

File: i18naddress/downloader.py [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+            'id', 'isoid', 'key', 'posturl', 'sub_isoids', 'sub_mores',
+            'sub_xrequires', 'sub_xzips', 'sub_zipexs']:
+        if unwanted in data:
+            del data[unwanted]
+    lang = data.pop('lang', None)
```
Comment (patrys) [inline i18naddress/downloader.py]: Also do not remove `lang` and `languages`.

Comment (mociepka) [inline i18naddress/downloader.py]: I will leave all data maybe someone will need them.

File: setup.py [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        pass
+
+    def run(self):
+        download()
+
+setup(
+    name='google-i18n-address',
+    author='Mirumee Software',
+    author_email='hello@mirumee.com',
+    description='Address valdation helpers wrapped in Google i18n API.',
```
Comment (patrys) [inline setup.py]: Address validation helpers for Google's i18n address database

**Pull request merged**